pkg/cloudprovider/aws/fleet/launchtemplate.go

```diff
@@ -55,25 +57,42 @@ type LaunchTemplateProvider struct {
 	clientSet               *kubernetes.Clientset
 }
 
-func (p *LaunchTemplateProvider) Get(ctx context.Context, cluster *v1alpha1.ClusterSpec) (*ec2.LaunchTemplate, error) {
-	if launchTemplate, ok := p.launchTemplateCache.Get(cluster.Name); ok {
+// Translate architecture into AWS-recognized name.
+func normalizeArchitecture(architecture cloudprovider.Architecture) string {
+	switch architecture {
+	case cloudprovider.ArchitectureAarch64:
+		return "arm64"
+	default:
+		// TODO: handle other architectures
+		return "x86_64"
+	}
+}
+
+func launchTemplateName(clusterName string, arch string) string {
```

Comment:
nit: I might just do this inline, since it takes the same arguments. You're gonna find yourself continually wiring, here.

Reply:
Also -- we probably want to put provisionername(space) into the launch template name

Reply:
Hmm.. if that's the case it seems like the launch template name should just be passed in somehow? Otherwise this LaunchTemplateProvider seems like it knows about too many things?

Reply:
Sure. Just cut a ticket for it

Reply:
done (273)